.circleci/config.yml

```
@@ -0,0 +1,19 @@
+version: 2.1
+
+jobs:
+  build-and-test:
+    docker:
+      - image: circleci/node:latest
```

```suggestion
      - image: circleci/node:12.13.0
```

That's the version we're using, and what we should be testing against. That version is documented in `.node-version`.

---

We probably want to build these libraries against a matrix of different versions of Node.js. So whatever the current LTS is, and the latest version, for example. Not a requirement for this PR, though.